doc/releases/changelog-dev.md

```diff
@@ -108,8 +86,76 @@
          [ 5.3798613 , -1.01459396,  0.        ]]))
   ```
 
-* New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
-  [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
+* Perform quantum chemistry calculations with two new basis sets: `6-311g` and `CC-PVDZ`.
+  [(#3279)](https://github.com/PennyLaneAI/pennylane/pull/3279)
+
+  ```pycon
+  >>> symbols = ["H", "He"] 
+  >>> geometry = np.array([[1.0, 0.0, 0.0], [0.0, 0.0, 0.0]], requires_grad=False)
+  >>> charge = 1
+  >>> basis_names = ["6-311G", "CC-PVDZ"] 
+  >>> for basis_name in basis_names:
+  ...     mol = qchem.Molecule(symbols, geometry, charge=charge, basis_name=basis_name)
+  ...     print(qchem.hf_energy(mol)())
+  [-2.84429531] 
+  [-2.84061284]
+  ```
+
+<h4>Gradient stuff</h4>
+
+* A new gradient transform, `qml.gradients.spsa_grad`, that is based on the idea of SPSA is now available.
+  [#3366](https://github.com/PennyLaneAI/pennylane/pull/3366)
+
+  This new transform allows users to compute a single estimate of a quantum gradient using simultaneous perturbation of parameters and a stochastic approximation. A QNode that takes, say, an argument `x`, the approximate gradient can be computed as follows.
+
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=2)
+  >>> x = pnp.array(0.4, requires_grad=True)
+  >>> @qml.qnode(dev)
+  ... def circuit(x):
+  ...     qml.RX(x, 0)
+  ...     qml.RX(x, 1)
+  ...     return qml.expval(qml.PauliZ(0))
+  >>> grad_fn = qml.gradients.spsa_grad(circuit, h=0.1, num_directions=1)
+  >>> grad_fn(x)
+  array(-0.38876964)
+  ```
+
+  The argument `num_directions` determines how many directions of simultaneous perturbation and, therefore, the number of circuit evaluations are used up to a prefactor. See the [SPSA gradient transform documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.gradients.spsa_grad.html) for details. Note: The full SPSA optimizer is already available as `qml.SPSAOptimizer`.
+
+<h4>Custom measurement processes</h4>
```

Comment:
I think it would really help to have some verbose code examples here 🙂. These entries still need to be distilled down to their core user-facing functionality, but we can address this in feature freeze week

Reply:
Sure!